lib/pbench/cli/server/report.py

```diff
@@ -435,13 +452,15 @@ def summarize_dates(query: Query, options: dict[str, Any]):
         if date >= start and date < end:
             in_range += 1
 
-    click.echo(
-        f"  {in_range:,d} since {start:%Y-%m-%d %H:%M} until {end:%Y-%m-%d %H:%M}"
-    )
-    click.echo(f"    {this_year:,d} in year {year:%Y}")
-    click.echo(f"    {this_month:,d} in month {month:%B %Y}")
-    click.echo(f"    {this_week:,d} in week {week:%B %d} to {day:%B %d}")
-    click.echo(f"    {this_day:,d} on {day:%d %B %Y}")
+    click.echo(f"  {in_range:,d} from {start:%Y-%m-%d %H:%M} to {end:%Y-%m-%d %H:%M}")
```

Comment:
Do you really want this printed if the user specified neither `since` nor `until`?  (Alternately, wouldn't it be better if, instead of printing the default values of `start` and/or `end` when they are unspecified, we printed the oldest and newest date values found in the data?)

Reply:
I'd briefly considered picking out the earliest and actual latest date, something that might be useful regardless of the range -- it seemed more trouble than it was worth. Then again, I have been tempted several times, and since you mention it as well, perhaps it's worth some effort after all.